test/x3/linker_error_test/example.cpp

```diff
@@ -0,0 +1,8 @@
+#include "example_def.hpp"
```

Comment:
And again... coding style.

Reply:
Do you mean the copyright again?  Surely the actual #include is OK.